modules/learn/pages/clusters-and-availability/xdcr-enable-crossclusterversioning.adoc

```diff
@@ -0,0 +1,65 @@
+= XDCR enableCrossClusterVersioning
+:description: pass:q[Enabling the bucket property `enableCrossClusterVersioning` allows XDCR to add metadata to each replicated document.]
+
+[abstract]
+{description}
+
+
+Enabling the bucket property `enableCrossClusterVersioning` for all buckets in the replication topology is a pre-requisite for some XDCR features. The bucket property `enableCrossClusterVersioning` cannot be disabled once it is set to `true`. Therefore, you must understand all the pre-requisites and the full effects of setting `enableCrossClusterVersioning` to `true` before enabling it.
+
+When you set the bucket property `enableCrossClusterVersioning` (ECCV) to `true`, for each document processed by XDCR, XDCR stores additional metadata for the document in the extended attributes. This metadata is also called Hybrid Logical Vector (HLV), which is a set of Hybrid Logical Clock (HLC) information.
+
+[#dependent-xdcr-features]
+== Dependent XDCR features
+
+This section lists feature settings on which replication related XDCR-mobile interoperability depends.
```

Comment:
Please remove this sentence.  Now that I'm looking at this page, I don't think that we should repeat the same information that you already have in the "XDCR Active-Active with Sync Gateway" page.

We should say:
The bi-directional, active-active replication with Sync Gateway 4.0+ and XDCR requires enabling Cross Cluster Versioning.  For more information, including important limitations, see xref:learn:clusters-and-availability/xdcr-active-active-sgw.adoc[XDCR Active-Active with Sync Gateway].